src/controllers/file.controller.js

```diff
@@ -0,0 +1,34 @@
+const httpStatus = require('http-status');
+const catchAsync = require('../utils/catchAsync');
+const multer = require('multer');
+const middlewareUpload = require('../middlewares/upload');
+const config = require('../config/config');
+
```

Comment:
`const logger = require('../config/logger');
`
Add the logger to show the error in your server logs as well. I actually found an issue below.